app/views/layouts/stories.html.erb

```diff
@@ -2,13 +2,69 @@
 <html lang="en" class="govuk-template">
     <%= render "sections/head" %>
     <%= analytics_body_tag class: "govuk-template__body govuk-body", data: { controller: "video link", target: "link.content" }, id: "body" do %>
-        <%= render "sections/header" %>
-        <main class="content container-1000" role="main" id="main-content">
-            <%= yield %>
-        </main>
-        <%= render "sections/footer" %>
-        <%= render "components/videoplayer" %>
-        <%= render "sections/cookie-acceptance" %>
+      <%= render "sections/header" %>
+      <main class="content container-1000" role="main" id="main-content">
+        <% if @front_matter.dig("build_layout_from_frontmatter") %>
+
+          <div class="content-wrapper">
+            <div class="content__left">
+              <article class="story">
+                <%= link_to(@front_matter["backlink_text"], @front_matter["backlink"], class: "backlink backlink--top") %>
+
+                <%= tag.h1(@front_matter["title"]) %>
+
+                <header class="story__header">
+                  <%= image_tag(@front_matter["image"], width: 200, height: 200, alt: story_image_alt(@front_matter.dig("story", "teacher"))) %>
+                  <div class="story__header__label">
+                    <%= story_heading(@front_matter) %>
+                  </div>
+                </header>
+
+                <% if youtube_url = @front_matter.dig("story", "video") %>
+                  <%= youtube(youtube_url) %>
+                <% end %>
+
+                <%= yield %>
+
+                <% if @front_matter["more_information"] %>
+                  <%= link_to(@front_matter.dig("more_information", "text"), @front_matter.dig("more_information", "link"), class: "git-link") %> <%= fas_icon "chevron-right" %>
+                <% end %>
+              </article>
+            </div>
+          </div>
+
+          <% if @front_matter.dig("more_stories").present? %>
```

Comment:
Likewise this could become `<%= render 'more_stories' if @front_matter.dig("more_stories).present? %>`